packer/main.pkr.hcl

```diff
@@ -76,7 +79,10 @@ build {
 
   provisioner "shell" {
     scripts = [
+      "./scripts/download_node_exporter.sh",
+      "./scripts/create_node_exporter_service.sh",
```

Comment:
Move download_node_exporter to packer while building image.
If done on TF -> Issue: no connection from github server.